Assets/__Scripts/UI/SongInfoEditUI.cs

```diff
@@ -285,10 +292,15 @@ private IEnumerator LoadAudio(bool useTemp = true)
         Debug.Log("Loading audio");
         if (File.Exists(fullPath))
         {
-            if (audioPath.text.ToLower().EndsWith("ogg") || audioPath.text.ToLower().EndsWith("egg"))
+
+            var extension = audioPath.text.Contains(".") ? Path.GetExtension(audioPath.text.ToLower()).Replace(".", "") : "";
+
+
+            if (!string.IsNullOrEmpty(extension))
             {
                 Debug.Log("Lets go");
-                UnityWebRequest www = UnityWebRequestMultimedia.GetAudioClip($"file:///{Uri.EscapeDataString($"{fullPath}")}", AudioType.OGGVORBIS);
+                var audioType = ExtensionToAudio.ContainsKey(extension) ? ExtensionToAudio[extension] : AudioType.UNKNOWN;
```

Comment:
Would it be a good idea to allow users to continue into the editor if the audio type is unknown? While Unity *might* support a few other audio formats, BeatSaver only supports `.ogg`, `.egg`, and `.wav` (I checked the source code). Furthermore, unsupported audio formats will show up as no audio at all, and I can guarantee you that it would be reported as a bug sometime in the future.

Reply:
I was unsure of this and assumed that might be a possibility. Thanks for confirming, will be taken into account